qubes_config/global_config.glade

```diff
@@ -1758,7 +1763,131 @@
                           </object>
                           <packing>
                             <property name="left-attach">1</property>
-                            <property name="top-attach">19</property>
+                            <property name="top-attach">20</property>
+                          </packing>
+                        </child>
+                        <child>
+                          <object class="GtkBox">
+                            <property name="visible">True</property>
+                            <property name="can-focus">False</property>
+                            <property name="orientation">vertical</property>
+                            <child>
+                              <object class="GtkLabel">
+                                <property name="visible">True</property>
+                                <property name="can-focus">False</property>
+                                <property name="label" translatable="yes">Preload disposables:</property>
```

Comment:
Do we want "Preload disposables", or maybe more explicitly "Preload disposable qubes"?